app/models/ui/OutputFormat.scala

```diff
@@ -60,12 +62,26 @@ case object Txt extends OutputFormat with Name {
   val name: String = "text"
 }
 
+case object DateStandard extends OutputFormat with Name {
```

Comment:
we could make the output format "d MMMM uuuu" 
e.g. [label:DueDate:d MMMM uuuu] 
or    [label:DueDate:date(d MMMM uuuu)]
which would fit in the Regex,  be supported by DateFormatter and give guidance writers more flexibility

Reply:
let's discuss as not sure we want designers to have to know the formats we use